db/migrate/20220623165020_create_projects.rb

```diff
@@ -0,0 +1,10 @@
+class CreateProjects < ActiveRecord::Migration[7.0]
+  def change
+    enable_extension "pgcrypto" unless extension_enabled?("pgcrypto")
```

Comment:
This is in the schema now, so I don't think we need it https://github.com/DFE-Digital/dfe-complete-conversions-transfers-and-changes/blob/58e5f827bfe08570ca8312d32f93521164eb0ba2/db/schema.rb#L15